prometheus_client/gc_collector.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/python
+
+from __future__ import unicode_literals
+
+import gc as gc_module
```

Comment:
There's no "gc" import you're shadowing here, so no need to rename.

Reply:
okay, I thought it made it a bit clearer when default the gc passed to the __init__ (which I needed for the tests)